.github/workflows/golangci-lint.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: '>=1.20'
+          go-version: '1.22'
```

Comment:
Oh I didn't realize this was happening! I've been post-ponning updating my dev env...

And we'll need to communicate this to validators.

Reply:
I'll add this requirement to the Docs section.